modules/widgets/social-icons.php

```diff
@@ -512,6 +512,106 @@ public function get_supported_icons() {
 				'icon'  => 'feed',
 				'label' => __( 'RSS Feed', 'jetpack' ),
 			),
+			array(
+				'url'   => '/feeds/',
+				'icon'  => 'feed',
+				'label' => __( 'RSS Feed', 'jetpack' ),
+			),
+			array(
+				'url'   => 'format=RSS',
+				'icon'  => 'feed',
+				'label' => __( 'RSS Feed', 'jetpack' ),
+			),
+			array(
+				'url'   => '/rss', // Used by Tumblr
+				'icon'  => 'feed',
+				'label' => __( 'RSS Feed', 'jetpack' ),
+			),
+            array(
```

Comment:
You seem to have some spaces in there instead of tabs. I would recommend setting up your IDE as recommended here to get recommendations to solve all those errors at once:
https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#use-php-codesniffer-and-eslint-to-make-sure-your-code-respects-coding-standards

Reply:
Ack, sorry, I'll fix that!

Reply:
I've pushed a fix for those spaces instead of tabs now.